tests/foreman/ui/test_audit.py

```diff
@@ -4,14 +4,12 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Medium
 
-:CaseComponent: UI
+:CaseComponent: AuditLog
 
 :TestType: Functional
 
-:CaseImportance: Low
-
```

Comment:
Looks like majority of tests here will have `:CaseImportance: Medium`. Don't you want to set Medium as default on module level, and only overwrite it in one test that is `Low`? Instead of removing value here and setting it for each test individually?